manifests/m/Mozilla/Firefox/100.0/Mozilla.Firefox.installer.yaml

```diff
@@ -16,783 +16,7553 @@ FileExtensions:
+    ProductCode: '{1294A4C5-9977-480F-9497-C0EA1E630130}'
+    AppsAndFeaturesEntries:
+    - Publisher: Mozilla
+      DisplayVersion: "100.0"
+      DisplayName: Mozilla Firefox (x86 bg)
+      ProductCode: Mozilla Firefox 100.0 (x86 bg)
+    - Publisher: Mozilla
+      DisplayVersion: "100.0"
+      DisplayName: Mozilla Maintenance Service
+      ProductCode: MozillaMaintenanceService
+  - InstallerLocale: my
+    Architecture: x64
+    InstallerType: exe
+    Scope: machine
+    InstallerUrl: https://download-installer.cdn.mozilla.net/pub/firefox/releases/100.0/win64/my/Firefox%20Setup%20100.0.exe
+    InstallerSha256: 936B730B72DC649F1478E20B74ACC6B8B87E9E7CB409770F54EDE2AF20382631
+    InstallerSwitches:
+      SilentWithProgress: /S
+      Silent: /S
+    AppsAndFeaturesEntries:
+    - Publisher: Mozilla
+      DisplayVersion: "100.0"
+      DisplayName: Mozilla Firefox (x64 my)
+      ProductCode: Mozilla Firefox 100.0 (x64 my)
+    - Publisher: Mozilla
+      DisplayVersion: "100.0"
+      DisplayName: Mozilla Maintenance Service
+      ProductCode: MozillaMaintenanceService
+  - InstallerLocale: my
+    Architecture: x64
+    InstallerType: wix
+    Scope: machine
+    InstallerUrl: https://download-installer.cdn.mozilla.net/pub/firefox/releases/100.0/win64/my/Firefox%20Setup%20100.0.msi
+    InstallerSha256: 4810C56FC8393DBC70D4394A265A37841DB93A203FC851488FA5E12BB4DD4E57
+    ProductCode: '{1294A4C5-9977-480F-9497-C0EA1E630130}'
+    AppsAndFeaturesEntries:
+    - Publisher: Mozilla
+      DisplayVersion: "100.0"
+      DisplayName: Mozilla Firefox (x64 my)
+      ProductCode: Mozilla Firefox 100.0 (x64 my)
+    - Publisher: Mozilla
+      DisplayVersion: "100.0"
+      DisplayName: Mozilla Maintenance Service
+      ProductCode: MozillaMaintenanceService
+  - InstallerLocale: my
+    Architecture: arm64
+    InstallerType: exe
+    Scope: machine
+    InstallerUrl: https://download-installer.cdn.mozilla.net/pub/firefox/releases/100.0/win64-aarch64/my/Firefox%20Setup%20100.0.exe
+    InstallerSha256: 26EA8D318301F20B49D4B9A3D151F1F97494CE47F420BF2ED35B7244A9A15C4F
+    InstallerSwitches:
+      SilentWithProgress: /S
+      Silent: /S
+  - InstallerLocale: my
+    Architecture: x86
+    InstallerType: exe
+    Scope: machine
+    InstallerUrl: https://download-installer.cdn.mozilla.net/pub/firefox/releases/100.0/win32/my/Firefox%20Setup%20100.0.exe
+    InstallerSha256: 0CCF2C97034A0B05B78B29EB3FBD74A5C570419F3F62BD9281C902A6410F5E11
+    InstallerSwitches:
+      SilentWithProgress: /S
+      Silent: /S
+    AppsAndFeaturesEntries:
+    - Publisher: Mozilla
+      DisplayVersion: "100.0"
+      DisplayName: Mozilla Firefox (x86 my)
+      ProductCode: Mozilla Firefox 100.0 (x86 my)
+    - Publisher: Mozilla
+      DisplayVersion: "100.0"
+      DisplayName: Mozilla Maintenance Service
+      ProductCode: MozillaMaintenanceService
+  - InstallerLocale: my
```

Comment:
Is this a valid InstallerLocale that winget knows how to deal with?

Reply:
After a very brief search, I only see MY as the region code for Malaysia.  I didn't find a language code MY.

However, I don't think that locales need to be "real" to work.  They just have to be [BCP-47](https://docs.microsoft.com/en-us/windows/win32/api/bcp47mrm/nf-bcp47mrm-iswellformedtag?msclkid=097f09f9d16611ec9c2cbe70208af896) format, and this is.  It isn't likely to match with any user's system settings, but if they said `--locale my` it would probably select it.